Mathlib/Algebra/Algebra/Operations.lean

```diff
@@ -14,6 +14,7 @@ import Mathlib.Data.Finset.Pointwise
 import Mathlib.Data.Set.Semiring
 import Mathlib.Data.Set.Pointwise.BigOperators
 import Mathlib.GroupTheory.GroupAction.SubMulAction.Pointwise
+import Mathlib.RingTheory.NonZeroDivisors
```

Comment:
is there not a better file for this? I think we're encouraged to avoid import creep. what does `#find_home` suggest?

Reply:
The problem is that `Mathlib.RingTheory.NonZeroDivisors` should not live in `RingTheory`. It only imports (and deals with) group theory.

Reply:
Yeah, we should probably move it. Even the doc says "In this file we define the submonoid `nonZeroDivisors` and `nonZeroSMulDivisors` of a `MonoidWithZero`.".

Reply:
This is now done, right?

Reply:
Yes. I updated the PR.